scripts/oneoff/export-service-categories.py

```diff
@@ -96,22 +97,26 @@ def get_categories(lot_name, service_data):
         'support': support_services
     }
     headers = [
-        'Supplier ID', 'DUNS Number', 'Supplier Name', 'Reseller?', 'Service Name', 'Organisation Size', 'Categories',
+        'Supplier ID', 'DUNS Number', 'Supplier Name', 'Reseller?', 'Service Name',
+        'Service Description', 'Organisation Size', 'Categories',
         '\n'
     ]
     for lot, services_in_lot in lots.items():
-        with open(os.path.join(OUTPUT_DIR, f'{lot}-categories-{version}.tsv'), 'w') as f:
-            f.write('\t'.join(headers))
+        with open(os.path.join(OUTPUT_DIR, f'{lot}-categories-{version}.csv'), 'w') as f:
```

Comment:
Python [docs](https://docs.python.org/3.6/library/csv.html#id2) state that `open` should have the `newline=''` option.

```suggestion
        with open(os.path.join(OUTPUT_DIR, f'{lot}-categories-{version}.csv'), 'w',, newline='') as f:
```

Reply:
Now added